GVFS/GVFS.Common/Enlistment.cs

```diff
@@ -35,7 +35,7 @@ public abstract class Enlistment
             else
             {
                 GitProcess.Result originResult = gitProcess.GetOriginUrl();
-                if (originResult.HasErrors)
+                if (originResult.ExitCodeIsFailure)
                 {
                     if (originResult.Errors.Length == 0)

```

Comment:
`if (!originResult.StderrContainsErrors())` instead of `if (originResult.Errors.Length == 0)`?